### ISSUE TITLE
Some InvocationExpressions are mutated twice

### ISSUE THREAD

**Issue by Mobrockers** [stryker-mutator/stryker-net#395]
![image](https://user-images.githubusercontent.com/3595932/54019396-60b14280-418b-11e9-8623-9fe20caa3cb8.png)

**Comment by Mobrockers** [comment]
All those mutations are double

**Comment by dupdob** [comment]
Well, let me look into this one

**Comment by dupdob** [comment]
I just ran the code and I got this result:
```csharp
public static string ChainCalls(string check)
{
   string check = Stryker.MutantControl.IsActive(10)?"Stryker was here!":"";
   return check.Replace(Stryker.MutantControl.IsActive(11)?"":"ab", Stryker.MutantControl.IsActive(12)?"":"cd")
      .Replace(Stryker.MutantControl.IsActive(13)?"":"12", Stryker.MutantControl.IsActive(14)?"":"34")
                .PadLeft(12)
      .Replace(Stryker.MutantControl.IsActive(15)?"":"12", Stryker.MutantControl.IsActive(16)?"":"34");
}
```

Can you dump the related mutated code?

**Comment by Mobrockers** [comment]
This is the file that resulted in double mutations.

```
namespace Test
{
    public class ExampleBug
    {
        public string ExampleBugMethod()
        {
            string someString = "";
            return someString.Replace("ab", "cd")
                .Replace("12", "34")
                .PadLeft(12)
                .Replace("12", "34");
        }
    }
}
```

This change worked on my system to remove the double mutations:
https://github.com/stryker-mutator/stryker-net/pull/396/files

I am however not confident that my testsize is large enough to see if this causes any side effects in other statements.

**Comment by dupdob** [comment]
I am afraid this change is aggressive. I check your sample and see what happens

**Comment by dupdob** [comment]
Ok, I reproduce the issue. I should be able to fix it then.

**Comment by dupdob** [comment]
By the way, your fix does not pass unit tests.

**Comment by Mobrockers** [comment]
Yes I know. I just pushed a quick fix that fixed the problem in my case as an example.

The full source code is the integrationtest project netcore2_1 in the mutation html report branch here:

https://github.com/stryker-mutator/stryker-net/tree/481c5e140541c85f1b33e46b8a6ba9cb79813cd9/integrationtests/TargetProjects/NetCore2_1/ExampleProject/String

I simply added the file there and removed the string mutation exclusion here:
https://github.com/stryker-mutator/stryker-net/blob/481c5e140541c85f1b33e46b8a6ba9cb79813cd9/integrationtests/TargetProjects/NetCore2_1/ExampleProject.XUnit/stryker-config.json

As far as I know I have not changed anything related to mutating in that branch. The branch was updated with your latest PR.
